Build/EasyNetQ.proj

```diff
@@ -6,20 +6,26 @@
         <OutputDir>$(BaseDir)\bin</OutputDir>
```

Comment:
With my changes in #644 , you can undo your changes to EasyNetQ.proj because they will conflict each other and I've updated the project to test using xUnit.